### ISSUE TITLE
AI and Machine Learning - Presentation of accessible outputs

### ISSUE THREAD

**Issue by joshueoconnor** [w3c/ai-accessibility#7]
#Suggestion ' it also is important to consider how accessible these outputs will be presented [and how this will effect the user experience of someone with a disability or who is a user of Assistive Technology], and if machine learning algorithm may address broader accessibility issues in everyday tasks. ‘

**Comment by sehollier** [comment]
Agreed, this should be fleshed out. For example, the recent Talkback update on Android provides user choice on whether alt text that is already present is used or generative AI alt text is used, and also provides the opportunity for multiple attempts on AI alt text results. However, it never indicates the accuracy of these alt texts unlike in Microsoft 365, where Microsoft does acknowledge its view on confidence levels, i.e. low confidence, high confidence of its alt text. That being said, sometimes its high confidence is still very wrong, so this all needs to be explained in this guidance.

Agree with the suggested approach but would still like to differentiate between the user experience of AI and professionals tasked with using AI to address accessibility issues.
